electricitymap/contrib/config/reading.py

```diff
@@ -0,0 +1,35 @@
+from typing import Any, Dict, List, NewType, Tuple
+
+import yaml
+
+from electricitymap.contrib.config.config_types import ZoneKey
+from electricitymap.contrib.config.constants import EXCHANGE_FILENAME_ZONE_SEPARATOR
+
+
+def read_defaults(config_dir) -> Dict[str, Any]:
+    """Reads the defaults.yaml file."""
+    defaults_path = config_dir.joinpath("defaults.yaml")
+    return yaml.safe_load(open(defaults_path, encoding="utf-8"))
```

Comment:
Here the return type don't match the return signature. We probably need to validate the type first with `isinstance`.

Reply:
So, yaml.safe_load generally should return a dict with str keys, but is the issue here just that it's not guaranteed to? Should I add something like the following?

```
d = yaml.safe_load(open(defaults_path, encoding="utf-8"))
assert isinstance(d, dict)
return d
```